src/site/antora/modules/ROOT/pages/manual/layouts.adoc

```diff
@@ -29,10 +29,20 @@ This page will try to answer following questions:
 
 This section introduces you to some common concerns shared by almost all <<collection,predefined layouts>> that you need to be aware of while using them.
 
+[#structured-logging]
+=== Structured logging
+
+Log4j strives to provide top of the class support for *structured logging*.
+To create an end-to-end experience, it provides several xref:manual/messages.adoc#collection-structured[structured message types] along with layouts supporting structured logging.
+
+include::partial$manual/structured-logging.adoc[]
```

Comment:
I think we don't need to collapse the "Structured logging" definition here.

Let us start with the include and then continue with "Log4j strives to provide..."

Reply:
Fixed in e6f2cd98126de18f4db9e6e8b75439dca222f27f.